### DISCUSSION TITLE
I am learning the source codes, found the clippy tips.

### DISCUSSION THREAD

**Discussion by headironc** [Totodore/socketioxide#348] [Q&A] [answered]
<img width="1272" alt="Screenshot 2024-07-13 at 10 05 12" src="https://github.com/user-attachments/assets/7933736a-0cb9-42c7-a108-98cc14706112">

There are many similar tips.

**Comment by Totodore** [comment, marked as answer]
These are only in testing or benchmark code no ? 

Did you find anything in the main source code.

You are right though, I should run clippy for everything and not only the main source code.

**Reply by headironc** [reply]
Yes, only in testing and benchmark

**Reply by headironc** [reply]
Can i open a pr to modify these contents?

**Reply by Totodore** [reply]
sure
